hack/sample/mysql-celery/cluster.yaml

```diff
@@ -21,26 +21,27 @@ metadata:
   name: mc-cluster
 spec:
   executor: Celery
-  config:
-    airflow:
-      AIRFLOW_SOME_CONFIG: SomeValue
   redis:
     operator: False
   scheduler:
-    version: "1.10.2"
+    image: "docker.io/vedantm/docker-airflow"
```

Comment:
This needs to be changed to point to `quay.io/opendatahub/docker-airflow`, version `openshift`

Reply:
Fixed